CONTRIBUTING.md

```diff
@@ -1,8 +1,74 @@
-# Contributing
+# Contributing to UMF (Unified Memory Framework)
+
+- [Opening new issues](#opening-new-issues)
+- [Submitting Pull Requests](#submitting-pull-requests)
+    - [Code style](#code-style)
+    - [License](#license)
+    - [Sign your work](#sign-your-work)
+    - [Adding new dependency](#adding-new-dependency)
+    - [Extending Public API](#extending-public-api)
+
+Below you'll find instructions on how to contribute to UMF, either with code changes
```

Comment:
Using either with more than two alternatives makes me unease. Maybe rephrase it a bit:
```
Below you'll find instructions on how to contribute to UMF. This can be done in three ways:
- open a PR with code changes
- file an issue with a problem to solve
- file an issue with an enhancement proposal.
```